StreamChatClient_v3/Utils/Logger/Logger.swift

```diff
@@ -0,0 +1,212 @@
+//
+// Logger.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public var log: Logger {
+  LogConfig.logger
+}
+
+public struct LogConfig {
+  /// Identifier for the logger. Defaults to empty.
+  public static var identifier = ""
```

Comment:
nit: If it defaults to empty, maybe it should rather be `String? = nil`

Reply:
Gave it a thought, it's way easier to handle if it's empty